### ISSUE TITLE
spm_jobman issue

### ISSUE THREAD

**Issue by lawlessrd** [ALXXLARG/FCT#3]
Hi Lyuan, 

I'm having trouble getting the spm_jobman function to work on my version of the code. Would you mind checking out my repo and seeing if you can spot what I'm doing wrong? Here is the exact error I'm getting on line 179 of FCT main:

```
K>> spm_jobman('run',matlabbatch);
Item tools: No field(s) named
cat
Error using spm_jobman>fill_run_job (line 472)
No executable modules, but still unresolved dependencies or incomplete module inputs.

Error in spm_jobman (line 247)
        sts = fill_run_job('run', cjob, varargin{3:end});
```

This function works on your version of the code, but when I check the matlabbatch variable that you feed into spm_jobman it is exactly the same as mine. Let me know if I can provide any additional info.

**Comment by ALXXLARG** [comment]
Hello Dylan,

My notification settings had an issue and I saw your message yesterday. Sorry about that. I am checking your repo now and see if i could try to find the answer. Are you talking about the 'FCT main' under 'FCT/matlab/src/fct_main.m' in your repo? I just checked again and spm_jobman functions are on lines 211, 231 and 269. I am wondering which one had issues? 

Thank you for your help!

**Comment by lawlessrd** [comment]
It on line 211. Let me know if it is anything obvious. If you can't spot anything on first glance maybe we can set up a Zoom meeting sometime this week to take a look together.

Thanks!

**Comment by ALXXLARG** [comment]
Yes. It may be better if we could take a look together. I am available on Tuesday, Thursday and Friday. 

Thank you for your help!

**Comment by lawlessrd** [comment]
How about Thursday morning? Maybe 10 or 11?

**Comment by ALXXLARG** [comment]
Sure. See you then. 11 is fine.
